packages/grid/_modules_/grid/hooks/features/columns/useGridColumns.ts

```diff
@@ -318,8 +319,8 @@ export function useGridColumns(
     logger,
     apiRef,
     props.columns,
-    options.columnTypes,
-    options.checkboxSelection,
+    props.columnTypes,
```

Comment:
**Off-topic**. Would it make sense to use the "rootProps" terminology for these props and systematically?